### PR TITLE
Add a step to look inside mirroring traffic with tcpdump in istio-proxy

### PR DESCRIPTION
**Pull request by hiroshi-nasu**
This description would help Istio users to get more understanding of traffic mirroring.
Could you review and accept it if possible?
I am new here and would like to contribute to the docs.

### CODE REVIEW

**Comment by hiroshi-nasu** [comment]
I signed CLA.

**Comment by geeknoid** [comment]
To fix the CI failure, please rebase and push again. Sorry about that.

**Comment by hiroshi-nasu** [comment]
I rebased and pushed again. The CI test seems to be passed.

File: content/docs/tasks/traffic-management/mirroring/index.md [hunk 1]
```diff
@@ -251,6 +251,118 @@ log entries for `v1` and none for `v2`:
     127.0.0.1 - - [07/Mar/2018:19:26:44 +0000] "GET /headers HTTP/1.1" 200 361 "-" "curl/7.35.0"
     {{< /text >}}
 
+1. If you want to take a look inside the traffic, run the following commands on another console and send the traffic:
```
Comment (sdake) [inline content/docs/tasks/traffic-management/mirroring/index.md]: `If you want to examine traffic internals, run the following commands on another console:`.

Comment (sdake) [reply content/docs/tasks/traffic-management/mirroring/index.md]: the part about "send the traffic:" is vague - how  is it sent?

Comment (hiroshi-nasu) [reply content/docs/tasks/traffic-management/mirroring/index.md]: Thank you for your reviewing. I will modify the description as you wrote.
`If you want to examine traffic internals, run the following commands on another console:`

"Send the traffic" means to run a command to send the sleep pod traffic on the former console like the previous step.
I think the readers would understand the context without the sentence if they read the steps from the start.

Comment (hiroshi-nasu) [reply content/docs/tasks/traffic-management/mirroring/index.md]: I fixed in [dff2916](https://github.com/istio/istio.io/pull/3267/commits/dff2916128000757c978fd6367359b2854ef97db).

**Comment by hiroshi-nasu** [comment]
Thank you for the approval and merging!
All of it can be public.
I would contribute the doc improvement and advancement.

**Pull request merged**